wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md

```diff
@@ -14,15 +17,15 @@ Removing audio
 
 3) If the following shows up because the video is h.264 (h.264 is the best right now):
 
-![](CV_01.png "CV_01.png")
+   a) Press either (no is better unless the program crashes).
 
-a) Press either (no is better unless the program crashes).
+![](CV_01.png "CV_01.png")
```

Comment:
L22: Add a text inside ``[]`` to indicate which should be there when the image doesn't load.